pages/linux/repquota.md

```diff
@@ -0,0 +1,27 @@
+# repquota
+
+> Display a summary of existing file quotas for a filesystem.
+
+- Report on all existing quotas:
```

Comment:
In line with the suggested description below, this could be "Report stats for all quotas in use:"?